ocpp-wamp/src/main/kotlin/com/izivia/ocpp/wamp/server/impl/OcppWampServerApp.kt

```diff
@@ -253,3 +274,6 @@ class OcppWampServerApp(
         }
     }
 }
+
+private fun Throwable.isConnectionReset(): Boolean =
+    this is SocketException && this.message.equals("Connection reset")
```

Comment:
I've tested that with real life case with toxyproxy.
no unit tests with mock it doesn't make much sense